internal/services/automation/automation_variable_string_resource_test.go

```diff
@@ -130,6 +130,7 @@ resource "azurerm_automation_variable_string" "test" {
   automation_account_name = azurerm_automation_account.test.name
   description             = "This variable is created by Terraform acceptance test."
   value                   = "Hello, Terraform Complete Test."
+  encrypted               = true
```

Comment:
(as above) can we split this out to a separate testcase, as it changes the purpose of the test?

```suggestion
```